{"variants":[{"paths":["\/documentation\/asyncmoya\/responseerror\/detail"],"traits":[{"interfaceLanguage":"swift"}]}],"primaryContentSections":[{"declarations":[{"tokens":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"detail"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"}],"platforms":["macOS"],"languages":["swift"]}],"kind":"declarations"}],"kind":"symbol","schemaVersion":{"patch":0,"minor":3,"major":0},"sections":[],"identifier":{"url":"doc:\/\/AsyncMoya\/documentation\/AsyncMoya\/ResponseError\/detail","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/AsyncMoya\/documentation\/AsyncMoya","doc:\/\/AsyncMoya\/documentation\/AsyncMoya\/ResponseError"]]},"metadata":{"fragments":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"detail"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"roleHeading":"Instance Property","externalID":"s:9AsyncMoya13ResponseErrorV6detailSSvp","title":"detail","role":"symbol","modules":[{"name":"AsyncMoya"}],"symbolKind":"property"},"abstract":[{"text":"에러에 대한 상세 설명 메시지","type":"text"}],"references":{"doc://AsyncMoya/documentation/AsyncMoya/ResponseError":{"fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"ResponseError","kind":"identifier"}],"navigatorTitle":[{"text":"ResponseError","kind":"identifier"}],"kind":"symbol","url":"\/documentation\/asyncmoya\/responseerror","title":"ResponseError","role":"symbol","type":"topic","abstract":[{"type":"text","text":"서버로부터 반환된 에러 응답을 표현하는 모델입니다."}],"identifier":"doc:\/\/AsyncMoya\/documentation\/AsyncMoya\/ResponseError"},"doc://AsyncMoya/documentation/AsyncMoya/ResponseError/detail":{"fragments":[{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"detail","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"}],"abstract":[{"type":"text","text":"에러에 대한 상세 설명 메시지"}],"type":"topic","kind":"symbol","title":"detail","url":"\/documentation\/asyncmoya\/responseerror\/detail","identifier":"doc:\/\/AsyncMoya\/documentation\/AsyncMoya\/ResponseError\/detail","role":"symbol"},"doc://AsyncMoya/documentation/AsyncMoya":{"url":"\/documentation\/asyncmoya","role":"collection","kind":"symbol","identifier":"doc:\/\/AsyncMoya\/documentation\/AsyncMoya","abstract":[{"type":"text","text":"Moya를 위한 현대적인 async\/await, Combine, RxSwift 확장 라이브러리"}],"title":"AsyncMoya","type":"topic"}}}